tasks/install_docker.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Setup DB volume for prometheus
+  docker_volume:
+    name: prometheus_db
```

Comment:
If volume is created for prometheus database then what is the behavior of `prometheus_db_dir`? Also why it is even changed in molecule test when it is obviously not used?

Reply:
prometheus_db_dir was used to build the docker volume with a bind option to set the volume in the directory given by prometheus_db_dir, due to a dead end with molecule I change the behavior of that part (build the volume) and only build a 'basic volume' that was compatible with the way molecule test stuff.. but that move was more about making things works than anything as I couldn't do the same for config files in a simple/usable way